source/features/next-scheduled-github-action.tsx

```diff
@@ -26,8 +27,16 @@ const getScheduledWorkflows = cache.function(async (): Promise<Record<string, st
 		}
 	`);
 
-	const workflows = workflowFiles?.entries;
-	if (!workflows) {
+	return workflowFiles?.entries ?? [];
+}, {
+	maxAge: {days: 1},
+	staleWhileRevalidate: {days: 10},
+	cacheKey: () => 'workflows:' + getRepo()!.nameWithOwner
+});
+
+const getScheduledWorkflows = async (): Promise<Record<string, string> | false> => {
+	const workflows = await getWorkflows();
+	if (workflows.length === 0) {
 		return false;
 	}
```

Comment:
You can drop this condition an just return `{}` instead of `false`. The only problem is that it complicates the later check from `!workflows` to `Object.keys(workflows).length === 0` or something like that. Thoughts?

```suggestion
```

Reply:
Exactly why I changed it. If you look at the last PR you would see it caused me issues.